bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -829,15 +829,14 @@ private static void validate(CloudBigtableConfiguration configuration, String ta
     checkNotNullOrEmpty(configuration.getZoneId(), "zoneId");
     checkNotNullOrEmpty(configuration.getClusterId(), "clusterId");
     if (tableId != null) {
-      checkNotNullOrEmpty(tableId, "tableid");
       if (BigtableSession.isAlpnProviderEnabled()) {
         try (BigtableConnection conn = new BigtableConnection(configuration.toHBaseConfig());
             Admin admin = conn.getAdmin()) {
-          Preconditions.checkState(admin.tableExists(TableName.valueOf(tableId)), "Table "
-              + tableId + " does not exist.  This dataflow operation could not be run.");
-        } catch (IOException | IllegalArgumentException | ExceptionInInitializerError e) {
-          LOG.error(String.format("Could not validate that the table exists: %s (%s)", e.getClass()
-              .getName(), e.getMessage()), e);
+          Preconditions.checkState(
+              admin.tableExists(TableName.valueOf(tableId)),
+              "Table " + tableId + " does not exist.  This dataflow operation could not be run.");
+        } catch (IOException e) {
```

Comment:
This means we never caught this exception before. Would this PR change anything?